chapters/publication.tex

```diff
@@ -99,91 +174,259 @@ \section{Collaborating on academic writing}
+A new user should, for example, be able to easily identify and remove
+any files created by the code so that they can be recreated quickly.
+They should also be able to quickly map all the outputs of the code
+to the locations where they are placed in the associated published material,
+such as ensuring that the raw components of figures or tables are clearly identified.
+For example, outputs should clearly correspond by name to an exhibit in the paper, and vice versa.
+(Supplying a compiling \LaTeX\ document can support this.)
+Code and outputs which are not used should be removed -- 
+if you are using GitHub, consider making them available in a different branch for transparency.
+
+\subsection{Releasing a replication package}
 
 If you are at this stage,
-all you need to do is find a place to publish your work!
-GitHub provides one of the easiest solutions here,
-since it is completely free for static, public projects
-and it is straightforward to simply upload a fixed directory
-and obtain a permanent URL for it.
-The Open Science Framework also provides a good resource,
-as does ResearchGate (which can also assign a permanent
-digital object identifier link for your work).
+all you need to do is find a place to publish your materials.
+This is slightly easier said than done,
+as there are a few variables to take into consideration
+and no global consensus on the best solution.
+The technologies available are likely to change dramatically
+over the next few years;
+the specific solutions we mention here highlight some current approaches
+as well as their strengths and weaknesses.
+GitHub provides one solution.
+Making your GitHub repository public
+is completely free for finalized projects.
+It can hold any file types,
+provide a structured download of your whole project,
+and allow others to look at alternate versions or histories easily.
+It is straightforward to simply upload a fixed directory to GitHub
+apply a sharing license, and obtain a URL for the whole package.
+
+However, GitHub is not ideal for other reasons.
+It is not built to hold data in an efficient way
+or to manage licenses or citations for datasets.
+It does not provide a true archive service --
+you can change or remove the contents at any time.
+A repository such as the Harvard Dataverse\sidenote{
+  \url{https://dataverse.harvard.edu}}
+addresses these issues, but is a poor place to store code.
+The Open Science Framework\sidenote{
+  \url{https://osf.io}}
+provides a balanced implementation
+that holds both code and data (as well as simple version histories),
+as does ResearchGate\sidenote{
+  \url{https://https://www.researchgate.net}}
+(both of which can also assign a permanent digital object identifier link for your work).
 Any of these locations is acceptable --
 the main requirement is that the system can handle
 the structured directory that you are submitting,
 and that it can provide a stable, structured URL for your project.
```

Comment:
_Any of these locations is acceptable --the main requirement is that the system can handle the structured directory that you are submitting, and that it can provide a stable, structured URL for your project._ 

Another very important main requirement is that the system also shows exactly what, if any, modifications you have done since the point of initial publishing. That is mentioned elsewhere, but important enough to also be included in this summary of main requirements.

Reply:
dd7cc864a220f7c2a144db4b1610a4e046435cb0